src/v/config/configuration.cc

```diff
@@ -2225,12 +2225,13 @@ configuration::configuration()
   , cloud_storage_azure_hierarchical_namespace_enabled(
       *this,
       "cloud_storage_azure_hierarchical_namespace_enabled",
-      "Whether or not Azure Hierarchical Namespaces are enabled on the "
-      "cloud_storage_azure_storage_account. If this property is not set, "
-      "cloud_storage_azure_shared_key must be set, and each node will try to "
-      "determine at startup if HNS is enabled. Setting this property to True "
-      "will disable the check and assume HNS is active. Setting to False will "
-      "disable the check and assume that HNS is not active.",
+      "Force Redpanda to use or not use \"Azure Data Lake Storage Gen2 "
+      "hierarchical namespaces\" compliant client. When this property is not "
+      "set, each node infers at startup if HNS is enabled. Setting this "
+      "property to True will disable the check and assume HNS is active. "
+      "Setting to False will disable the check and assume that HNS is not "
+      "active. This setting should be used only in emergencies where Redpanda "
+      "fails to detect the correct HNS status.",
```

Comment:
```suggestion
      "Force Redpanda to use or not use an \"Azure Data Lake Storage Gen2 "
      "hierarchical namespace\" compliant client. When this property is not "
      "set, each node infers at startup if HNS is enabled. When set to True, "
      "this property disables the check and assumes HNS is active. "
      "When set to False, this property disables the check and assumes HNS is not "
      "active. This setting should be used only in emergencies where Redpanda "
      "fails to detect the correct HNS status.",
```

Reply:
Applied this and replaced "HNS active" to "HNS enabled".